### ISSUE TITLE
How to evaluate model?

### ISSUE THREAD

**Issue by MuhammadAsadJaved** [eric612/MobileNet-YOLO#174]
I have train darknet_yolov3 on custom dataset.  Now i am running test.sh script to evaluate the model on the test dataset but I do not understand the results . can someone elaborate the results?  like where is  Precision , recall and mAP etc?  If it is not shown here then how can I get it?  what is this ouput of iou:, cat: , recall: 1 recall 75:  and loss = 4.4.39057 .  Is it the performance of model?  Can someone elaborate please? 

I1009 10:28:22.354635  8789 yolov3_layer.cpp:411] noobj: 0.00116894 obj: 0.625343 iou: 0.732791 cat: 0.917297 recall: 1 recall75: 0.52381 count: 1
I1009 10:28:22.357827  8789 yolov3_layer.cpp:411] noobj: 0.00119317 obj: 0.431886 iou: 0.719641 cat: 0.987395 recall: 0.922619 recall75: 0.509354 count: 3
I1009 10:28:22.369971  8789 yolov3_layer.cpp:411] noobj: 0.000346323 obj: 0.28554 iou: 0.664695 cat: 0.966271 recall: 0.969198 recall75: 0.251498 count: 4
I1009 10:28:22.434128  8789 solver.cpp:252] Iteration 220 (0.119206 iter/s, 83.8882s/10 iters), loss = 4.39057
I1009 10:28:22.434175  8789 solver.cpp:271]     Train net output #0: det_loss1 = 0.00302535 (* 1 = 0.00302535 loss)
I1009 10:28:22.434183  8789 solver.cpp:271]     Train net output #1: det_loss2 = 0.164056 (* 1 = 0.164056 loss)
I1009 10:28:22.434188  8789 solver.cpp:271]     Train net output #2: det_loss3 = 1.97582 (* 1 = 1.97582 loss)
I1009 10:28:22.434195  8789 sgd_solver.cpp:121] Iteration 220, lr = 1e-05

**Comment by MuhammadAsadJaved** [comment]
![model](https://user-images.githubusercontent.com/28862708/66450526-7709de80-ea8b-11e9-8db5-f513e51acd8b.png)
see the picture for better understanding.
